### PR TITLE
[pulsar-io]Add document for canal

### PR DESCRIPTION
**Pull request by tuteng**
### Documentation

  - Add document for canal

### CODE REVIEW

File: site2/docs/io-cdc.md [hunk 1]
```diff
@@ -8,139 +8,6 @@ sidebar_label: CDC Connector
...
-mysql> show tables;
-mysql> SELECT * FROM  products ;
-mysql> UPDATE products SET name='1111111111' WHERE id=101;
-mysql> UPDATE products SET name='1111111111' WHERE id=107;
-```
-
-- In above subscribe topic terminal tab, we could find that 2 changes has been kept into products topic.
+The CDC Source connector is built on top of [Debezium](https://debezium.io/) and [Canal](https://github.com/alibaba/canal). This connector stores all data into Pulsar Cluster in a persistent, replicated and partitioned way.
+This CDC Source are tested by using MySQL, and you could get more information regarding how it works at [this debezium link](https://debezium.io/docs/connectors/mysql/) or [this canal link](https://github.com/alibaba/canal/wiki).
+Regarding how Debezium works, please reference to [Debezium tutorial](https://debezium.io/docs/tutorial/). Regarding how Canal works, please reference to [Canal tutorial](https://github.com/alibaba/canal/wiki). It is recommended that you go through this tutorial first.
```
Comment (sijie) [inline site2/docs/io-cdc.md]: @tuteng can you link the tutorials to canal and debezium?

**Comment by sijie** [comment]
@tuteng can you also modify the documentation for version-2.3.0 as well?

**Comment by tuteng** [comment]
> @tuteng can you also modify the documentation for version-2.3.0 as well?

Is it edited on branch-2.3?

**Comment by sijie** [comment]
@tuteng https://github.com/apache/pulsar/tree/master/site2/website/versioned_docs/version-2.3.0

**Comment by tuteng** [comment]
> @tuteng it is in master: https://github.com/apache/pulsar/tree/master/site2/website/versioned_docs/version-2.3.0

@sijie Modify completed

**Pull request merged**